railties/lib/rails/generators/rails/app/templates/app/assets/javascripts/application.js.tt

```diff
@@ -11,8 +11,10 @@
 // about supported directives.
 //
 <% unless options[:skip_javascript] -%>
+<% if options[:javascript] -%>
 //= require <%= options[:javascript] %>
-//= require <%= options[:javascript] %>_ujs
+<% end -%>
```

Comment:
For users passing the `-j jquery` option this file will be like:

```
//= require jquery
//= require rails-ujs
```

So we are making rails-ujs the default even for jQuery users and not allowing people to chose another driver